### DISCUSSION TITLE
Tap tempo epic

### DISCUSSION THREAD

**Discussion by dcallus** [fillmaster/fillMaster#51] [General]
Hey @ninad-d could you please have a look in the branch called 'some-changes' please? https://github.com/fillmaster/fillMaster/tree/some-changes

It is passing the tests but it's not running. I'm getting an error saying:
Cannot read properties of undefined (reading 'timeSignature')

I basically moved the classes into a new folder. Did a bunch of ESLint stuff and re-implemented some missing functionality from the pattern class refactors. Things were pretty much working, but I've messed up somehow and I can't figure out why.

Also if there is any tidying up or things that you think are wrong please feel free to make some changes.

If you don't have time I'll look again in the next couple of days with a fresh mind.

thanks

**Comment by ninad-d** [comment]
will take a look at it.
